src/Apps/Collect/Components/Base/CollectArtworkGrid.tsx

```diff
@@ -86,6 +86,7 @@ class CollectArtworkGrid extends Component<Props, LoadingAreaState> {
                     <ArtworkGrid
                       artworks={artworks as any}
                       columnCount={columnCount}
+                      preloadImageCount={9}
```

Comment:
Or should _this_ be a default prop 😄

Reply:
It _could_ be. Right now I have the default as 6 which seems to cover most cases.